sherlock_project/resources/data.json

```
@@ -628,6 +628,13 @@
     "urlMain": "https://deviantart.com",
     "username_claimed": "blue"
   },
+  "DigitalSpy": {
+    "errorMsg": "The page you were looking for could not be found.",
+    "errorType": "message",
+    "url": "https://forums.digitalspy.com/profile/{}",
+    "urlMain": "https://forums.digitalspy.com/",
+    "username_claimed": "papita69"
```

Looks like a good add! The only issue is that this username no longer appears to be active. When testing with `blue` however, it works as expected.

Probably worked fine when you added it, the account just closed down.

```suggestion
    "username_claimed": "blue"
```

---

noted,